src/API/Management/Blacklists.php

```diff
@@ -15,7 +15,11 @@ public function getAll($aud)
     {
         $response = $this->httpClient->get('/blacklists/tokens?'.http_build_query(['aud' => $aud]));
 
-        return ResponseMediator::getContent($response);
+        if (200 === $response->getStatusCode()) {
```

Comment:
Yes I would do the codes mapping inside the `ResponseMediator`:

204 => success (should do nothing)
2xx, 4xx or 5xx => check for content type and parse if needed